multirun/def.bzl

```diff
@@ -60,9 +60,11 @@ def _multirun_impl(ctx):
             tag = tag,
             path = exe.short_path,
         ))
+    if ctx.attr.parallel:
+       print("'parallel' attribute is deprecated. Use 'jobs' instead.")
     instructions = struct(
         commands = commands,
-        parallel = ctx.attr.parallel,
+        jobs = ctx.attr.jobs,
```

Comment:
Will probably check in both go binary and in starlark. Just in case some poor soul (me) decided to just http_file the binary in the future without using the starlark wrapper.